### DISCUSSION TITLE
Getting real file paths in desktop builds

### DISCUSSION THREAD

**Discussion by Kixiron** [DioxusLabs/dioxus#351] [Q&A]
I need a file picker for my desktop app so I tried to use a file input, but the path it returns is `C:\fakepath\foo` like you'd get from a browser, is there a way to get the actual path of selected things?

**Comment by jkelleyrtp** [comment]
We should really be handling this in Dioxus but you will want to lean on https://github.com/saurvs/nfd-rs or another crate to spawn a native dialog.
